apps/rush-lib/src/cli/actions/BaseInstallAction.ts

```diff
@@ -64,7 +64,7 @@ export abstract class BaseInstallAction extends BaseRushAction {
     if (this.actionName === 'update') {
       warnAboutScriptUpdate = StandardScriptUpdater.update(this.rushConfiguration);
     } else {
-      StandardScriptUpdater.validate(this.rushConfiguration);
+//      StandardScriptUpdater.validate(this.rushConfiguration);
```

Comment:
why is this commented out?

Reply:
Good catch!


---
In reply to: [211060088](https://github.com/Microsoft/web-build-tools/pull/775#discussion_r211060088) [](ancestors = 211060088)